1_fetch/src/fetch_nwis.R

```diff
@@ -30,14 +30,110 @@ fetch_gw_uv <- function(target_name, gw_sites, start_date, end_date, param_cd) {
     write_feather(target_name)
 }
 
-convert_uv_to_dv <- function(target_name, gw_uv_data_fn) {
+convert_uv_to_dv <- function(target_name, gw_uv_data_fn, site_tz_xwalk) {
```

Comment:
Is this different from `fetch_nwis_historic.r`?

Reply:
Nope ... will address this in #74